src/NuGet.Clients/PackageManagement.VisualStudio/ScriptPackage.cs

```diff
@@ -70,5 +83,59 @@ private IEnumerable<IPackageAssemblyReference> GetAssemblyReferencesCore()
 
             return result;
         }
+
+        private List<ScriptPackageFile> GetFilesCore()
+        {
+            var result = new List<ScriptPackageFile>();
+            var reader = GetPackageReader(_installPath);
+
```

Comment:
Add a Debug.Assert for reader != null here, this seems too easy to get around right now